sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -153,6 +156,23 @@ def test_read(self):
     def test_no_subtransform_composite(self):
       raise unittest.SkipTest("BEAM-4781")
 
+    def test_external_transform(self):
+      # options = super(FlinkRunnerTest, self).create_options()
+      # options._all_options['shutdown_sources_on_final_watermark'] = True
+      # options._all_options['parallelism'] = 1
+      # options._all_options['streaming'] = True
+
+      self.create_options()
+      expansion_address = "localhost:" + str(FlinkRunnerTest.expansion_port)
+
+      with self.create_pipeline() as p:
+        res = (
```

Comment:
Probably we should make this a post-commit test instead of pre-commit.

Reply:
I'm not sure, all of these tests used to be PostCommit, but we moved them to PreCommit to have less PostCommit breakage. It might be good to have it in PreCommit for now.